src/bindings/python/src/openvino/runtime/__init__.py

```diff
@@ -67,6 +67,10 @@
 from openvino.runtime.ie_api import compile_model
 
 
+# Model Conversion API
+from openvino.runtime.convert import convert_model, InputCutInfo, LayoutMap
```

Comment:
I think we should put `convert_model` and other relative stuff into the separate directory. Now we are mixing mo stuff with ops, etc. Don't think this is good.
Moreover, it will have to split our space in codeowners file instead to review all runtime stuff.

Reply:
Done.